ui/pages/Project/components/FamilyTable/FamilyRow.jsx

```diff
@@ -0,0 +1,170 @@
+import TextFieldView from 'shared/components/panel/view-fields/TextFieldView'
+import ListFieldView from 'shared/components/panel/view-fields/ListFieldView'
+import { FAMILY_ANALYSIS_STATUS_LOOKUP } from 'shared/constants/familyAndIndividualConstants'
+import { getProject, updateFamilies } from 'redux/rootReducer'
+
+
+const mapDispatchToProps = (dispatch, ownProps) => {
+  return {
+    updateFamily: (values) => {
+      dispatch(updateFamilies({ families: [{ familyGuid: ownProps.family.familyGuid, ...values }] }))
+    },
+  }
+}
+
+
+const AnalysisStatus = ({ family, project, canEdit }) => {
+  const familyAnalysisStatus = (
+    (family.analysisStatus && FAMILY_ANALYSIS_STATUS_LOOKUP[family.analysisStatus]) ?
+      FAMILY_ANALYSIS_STATUS_LOOKUP[family.analysisStatus] :
+      {}
+  )
+  return (
+    <div style={{ whiteSpace: 'nowrap' }}>
+      <div style={{ display: 'inline-block', padding: '5px 15px 5px 0px' }}><b>Analysis Status: </b></div>
+      <Popup
+        trigger={<Icon name="play" style={{ color: familyAnalysisStatus.color }} />}
+        content={<div>Analysis Status:<br />{familyAnalysisStatus.name}</div>}
+      />
+      {familyAnalysisStatus.name}
+      {canEdit && project.canEdit &&
+        <a
+          style={{ paddingLeft: '15px' }}
+          href={`/project/${project.deprecatedProjectId}/family/${family.familyId}/edit`}
+        >
+          <Icon name="write" size="small" />
+        </a>
+      }
+    </div>
+  )
+}
+
+AnalysisStatus.propTypes = {
+  project: PropTypes.object.isRequired,
+  family: PropTypes.object.isRequired,
+  canEdit: PropTypes.bool,
+}
+
+const BaseAnalysedBy = ({ family, updateFamily, canEdit }) =>
+  <ListFieldView
+    isEditable={canEdit}
+    fieldName="Analysed By"
+    values={family.analysedBy.map(analysedByObj => `${analysedByObj.user.display_name} (${analysedByObj.date_saved})`)}
+    addItemUrl={`/api/family/${family.familyGuid}/update_analysed_by`}
+    onItemAdded={updateFamily}
+    confirmAddMessage="Are you sure you want to add that you analysed this family?"
+  />
+
+BaseAnalysedBy.propTypes = {
+  family: PropTypes.object.isRequired,
+  canEdit: PropTypes.bool,
+  updateFamily: PropTypes.func,
+}
+
+const AnalysedBy = connect(null, mapDispatchToProps)(BaseAnalysedBy)
+
+export const DESCRIPTION = 'description'
+export const ANALYSIS_STATUS = 'analysisStatus'
+export const ANALYSED_BY = 'analysedBy'
+export const ANALYSIS_NOTES = 'analysisNotes'
+export const ANALYSIS_SUMMARY = 'analysisSummary'
+export const INTERNAL_NOTES = 'internalCaseReviewNotes'
+export const INTERNAL_SUMMARY = 'internalCaseReviewSummary'
+
+const fieldRenderDetails = {
+  [DESCRIPTION]: { name: 'Family Description' },
+  [ANALYSIS_STATUS]: { component: AnalysisStatus },
+  [ANALYSED_BY]: { component: AnalysedBy },
+  [ANALYSIS_NOTES]: { name: 'Analysis Notes' },
+  [ANALYSIS_SUMMARY]: { name: 'Analysis Summary' },
+  [INTERNAL_NOTES]: { name: 'Internal Notes', internal: true },
```

Comment:
Editing internal notes doesn't appear to work on Case Review page. 
Edit form appears, can be submitted, and generates a request, but the entered text doesn't appear on the page (even after refreshing)

Reply:
Same thing for a few other Family-level fields: Analysis Notes, Analysis summary.

Reply:
Also, rich text formatting isn't getting saved:
![image](https://user-images.githubusercontent.com/6240170/40095454-1f40097e-589a-11e8-86a2-0cd0f048c18e.png)